packages/arco-lib/src/components/Table/Table.tsx

```diff
@@ -283,6 +296,9 @@ export const Table = implementRuntimeComponent({
     setColumns(
       cProps.columns!.map((column, i) => {
         const newColumn: ColumnProperty = { ...column };
+        if (newColumn.width === -1) {
+          Reflect.deleteProperty(newColumn, 'width');
```

Comment:
Since `newColumn.width` is checked below, do we have to delete it here? It is not good to mutate properties.

Reply:
changed